renderers/photoaudiohaptics.schema.json

```diff
@@ -25,12 +25,16 @@
                 "maximum": 1
             }
         },
-        "contourPoints": {
+        "contours": {
             "type": "array",
             "description": "An array containing an array of normalized coordinates representing a point on the segment or point on the bounding box.",
             "items": {
                 "$ref": "#/definitions/coordinates"
             }
+        },
+        
+        "entityType":{
```

Comment:
Could you add more documentation for this? A description is a must, and if this only takes a few values then it should probably just be enumerated.

Reply:
Sorry, I forgot to add the description! I'll also add a max length for the string